.rubocop.yml

```diff
@@ -2,3 +2,13 @@ inherit_gem:
   rubocop-govuk:
     - config/default.yml
     - config/rails.yml
+
+AllCops:
+  Exclude:
+    - 'bin/*'
+    - 'db/schema.rb'
+    - 'node_modules/**/*'
+    - 'config/application.rb'
+    - 'config/puma.rb'
+    - 'config/environments/*'
```

Comment:
Should we just exclude `config/environment.rb` here?

Reply:
I just got rid of this line - Rubocop still runs cleanly without it.